prospero-cli/src/main/java/org/wildfly/prospero/cli/commands/InstallCommand.java

```diff
@@ -37,65 +37,74 @@
 
 @CommandLine.Command(
         name = CliConstants.INSTALL,
-        sortOptions = false
+        sortOptions = false/*,
+        customSynopsis = "@|bold prospero install|@ @|fg(yellow) --fpl|@=@|italic <feature-pack>|@ @|fg(yellow) [OPTION]|@...\n"
```

Comment:
is the comment meant to stay here?

Reply:
No, just a forgotten bit. I think it's more handy to have all strings in resource bundles.